packages/swarm-components/src/Checkbox.jsx

```diff
@@ -20,15 +20,14 @@ const Checkbox = (props: Props): React.ReactElement<'label'> => {
 		>
 			<span
 				data-swarm-checkbox-field={checked ? 'checked' : 'unchecked'}
-				tabIndex="0"
-				role="checkbox"
+				role='checkbox'
 				aria-checked={checked}
 			>
 				{checked && (
 					<Icon shape="check" size="xs" color={disabled ? '#A2A2A2' : '#ffffff'} />
 				)}
 			</span>
-			<input type="checkbox" id={id} checked={checked} disabled={disabled} />
+			<input type="checkbox" id={id} checked={checked} disabled={disabled} onChange={onChange} />
```

Comment:
Hmm isn’t this change already in master? I did it on my Checkbox PR but even after this space wasn’t toggling the check box.
https://github.com/meetup/swarm-ui/blob/master/packages/swarm-components/src/Checkbox.jsx

Reply:
@jaclynj the `onChange` was in master but we were also forcing the focus with `tabIndex` to the `span` and not allowing it on the `input` so visually the box was focused while the browser was focused on the styled `span` rather than the `input`. The `tabIndex` was removed in this pr https://github.com/meetup/swarm-ui/pull/13/files#diff-10d336042a35c6a6395a71ea167c1393L45